docs/integrations/google/workspace/collect-logs.md

```diff
@@ -200,496 +198,3 @@ _sourceCategory=google*
 | count by email
 | sort by _count desc, email asc
 ```
```

Comment:
This removal is correct. We just need to point to the new C2C source page, currently at https://help.sumologic.com/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-workspace-alertcenter/